### PR TITLE
[GNA] Fix for GeminiLake detection

### PR DESCRIPTION
**Pull request by marcinkacprzak**
### Details:
 - Added Gna2DeviceVersion1_0 to GnaDeviceMap for GNL support

### Tickets:
 - **115717**

### CODE REVIEW

File: src/plugins/intel_gna/src/gna_plugin.cpp [hunk 1]
```diff
@@ -3,6 +3,7 @@
 //
 
 #define NOMINMAX
+#define PLOT
```
Comment (sirzabek) [inline src/plugins/intel_gna/src/gna_plugin.cpp:6]: Please remove.

File: src/plugins/intel_gna/tests/functional/limitations/layers_limit.cpp [hunk 2]
```diff
@@ -84,8 +84,8 @@ TEST_P(GNALayersLimit3XTest, CompareWithRefs) {
 
 std::map<std::string, std::string> common_config{{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_COMPACT_MODE", "NO"}};
 
-std::vector<std::map<std::string, std::string>> configs_20{{{"GNA_EXEC_TARGET", "GNA_TARGET_2_0"}},
-                                                           {{"GNA_COMPILE_TARGET", "GNA_TARGET_2_0"}}};
+std::vector<std::map<std::string, std::string>> configs_20{{{"GNA_EXEC_TARGET", "GNA_TARGET_1_0"}},
```
Comment (sirzabek) [inline src/plugins/intel_gna/tests/functional/limitations/layers_limit.cpp:87]: Why replace and not add GNA1.0?

File: src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/conv_low_precision.cpp [hunk 3]
```diff
@@ -128,15 +130,27 @@ TEST_P(ConvLowPrecisionTest, CompareWithRefs) {
...
-const vector<map<string, string>> configs_3_X = {
-    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_PRECISION", "I8"}, {"GNA_EXEC_TARGET", "GNA_TARGET_3_0"}},
-    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_PRECISION", "I16"}, {"GNA_EXEC_TARGET", "GNA_TARGET_3_0"}},
-    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_PRECISION", "I8"}, {"GNA_EXEC_TARGET", "GNA_TARGET_3_5"}},
-    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_PRECISION", "I16"}, {"GNA_EXEC_TARGET", "GNA_TARGET_3_5"}}};
+const vector<map<string, string>> configs_1 = {
+    {{"GNA_DEVICE_MODE", "GNA_AUTO"}, {"GNA_PRECISION", "I8"}, {"GNA_EXEC_TARGET", "GNA_TARGET_1_0"}},
+    {{"GNA_DEVICE_MODE", "GNA_AUTO"}, {"GNA_PRECISION", "I16"}, {"GNA_EXEC_TARGET", "GNA_TARGET_1_0"}},
+    {{"GNA_DEVICE_MODE", "GNA_AUTO"}, {"GNA_PRECISION", "I8"}, {"GNA_EXEC_TARGET", "GNA_TARGET_1_0_E"}},
+    {{"GNA_DEVICE_MODE", "GNA_AUTO"}, {"GNA_PRECISION", "I16"}, {"GNA_EXEC_TARGET", "GNA_TARGET_1_0_E"}},
```
Comment (sirzabek) [inline src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/conv_low_precision.cpp:137]: I don't think we want to test embedded targets here and below.

**Review by sirzabek — CHANGES_REQUESTED** [review]

File: src/plugins/intel_gna/tests/functional/limitations/layers_limit.cpp (hunk 2 above)
Comment (marcinkacprzak) [reply src/plugins/intel_gna/tests/functional/limitations/layers_limit.cpp]: Pushed by mistake. Reverted.

File: src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/conv_low_precision.cpp (hunk 3 above)
Comment (marcinkacprzak) [reply src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/conv_low_precision.cpp]: Removed

**Review by mryzhov — APPROVED** [review]

**Review by sirzabek — APPROVED** [review]

File: src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/conv_low_precision.cpp [hunk 4]
```diff
@@ -128,15 +130,21 @@ TEST_P(ConvLowPrecisionTest, CompareWithRefs) {
 const vector<InferenceEngine::Precision> netPrecisions = {InferenceEngine::Precision::FP16,
                                                           InferenceEngine::Precision::FP32};
 
-const vector<map<string, string>> configs_3_X = {
-    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_PRECISION", "I8"}, {"GNA_EXEC_TARGET", "GNA_TARGET_3_0"}},
-    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_PRECISION", "I16"}, {"GNA_EXEC_TARGET", "GNA_TARGET_3_0"}},
-    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_PRECISION", "I8"}, {"GNA_EXEC_TARGET", "GNA_TARGET_3_5"}},
-    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_PRECISION", "I16"}, {"GNA_EXEC_TARGET", "GNA_TARGET_3_5"}}};
+const vector<map<string, string>> configs_1 = {
+    {{"GNA_DEVICE_MODE", "GNA_AUTO"}, {"GNA_PRECISION", "I8"}, {"GNA_EXEC_TARGET", "GNA_TARGET_1_0"}},
```
Comment (sirzabek) [inline src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/conv_low_precision.cpp:134]: Why did you change the device mode?
